pages/Documents/ConversationalAI/Third-PartyBots/IBMWatsonAssistant/advanced-features.md

```diff
@@ -335,6 +335,18 @@ A demo of our WhatsApp map example with the above bot configuration (defined abo
 
 <img class="fancyimage" style="width:300px" src="img/watsonassistant/watson_v1_richcontent_demo.gif">
 
+### Receiving Last consumer message (Messaging Only)
+
+Third-Party bot now provides a way to add the last consumer message as a part of the welcome event (messaging only). When an ongoing conversation gets transferred to a new Agent or Skill, This enhancement will allow brands to respond to the last consumer message uttered as per their needs.
+
+The last consumer message is passed via the property `lastConsumerMessage` which is part of another property `lpEvent` that is sent with `context` information by Third-Party Bots. Thus, for Watson Assistant, we can leverage the [Context Variables](https://cloud.ibm.com/docs/assistant?topic=assistant-dialog-runtime-context#dialog-runtime-context-variables)
+
+An example of accessing `lastConsumerMessage` from that context variable can be seen in Figure 3.10.
+
+<img class="fancyimage" style="width:600px" src="img/ThirdPartyBots/watson_consumer-message-access-event.png">
+Figure 3.10 Displaying how to access the last consumer message using Context Variables
```

Comment:
if you added the Figure number make sure you adjust the following page figure numbers as well